practice-project-api/routes/calculateRoutes.ts

```diff
@@ -0,0 +1,34 @@
+import { Router, Request, Response } from "express";
+
+export const calculateRoute = Router();
+
+calculateRoute.post("/calculate", (req: Request, res: Response, next) => {
+
+  type calculatorOperation = '*' | '/' | '+' | '-';
+
+  interface Calculate {
+    firstNumber: number,
+    secondNumber: number,
+    operation: calculatorOperation,
+  };
+
+  const input: Calculate = {
+    firstNumber: req.body.firstNumber,
+    secondNumber: req.body.firstNumber,
```

Comment:
This should be `req.body.secondNumber`. Also, change to `secondValue`.

Reply:
It looks like I've messed some stuff up in trying to roll back changes and apply API changes to PP-8/UI changes to PP-7. Are there any situations where you might want to be working on multiple branches at once for a project but don't necessarily want to push changes on one branch before switching to another? Because I'm having trouble with that. Says I need to stash, but obviously I don't want to lose the work I've done.

Reply:
Usually the UI and API are in separate repos, so this problem can be avoided. You've kinda stacked both branches on top of one another here... which is an annoying issue but you can learn to rebase things based on certain commits.

Reply:
You really should just get 1 task working at a time... this branch doesn't seem to do what the task needs.